test/org/sagebionetworks/bridge/models/healthdata/HealthDataRecordTest.java

```diff
@@ -299,13 +305,15 @@ public void testSerialization() throws Exception {
         // start with JSON
         String jsonText = "{\n" +
                 "   \"createdOn\":\"2014-02-12T13:45-0800\",\n" +
+                "   \"createdOnTimeZone\":\"-0800\",\n" +
```

Comment:
Discussed offline. This is an artifact of how in DDB this is represented as epoch milliseconds, but we convert it to DateTime when returning it as JSON. We never read this from an API, and in the uploads, we always get it from createdOn, so there will never be a data discrepancy.

It may be confusing to show a timestamp in a timezone that's not the user-submitted one, but it's not a problem unique to Bridge, and I'm not sure how to fix it.